tests/conftest.py

```diff
@@ -1782,6 +1783,8 @@ def awscli_pod_fixture(request, scope_name):
     )
     awscli_pod_dict["metadata"]["name"] = awscli_pod_name
 
+    update_container_with_mirrored_image(awscli_pod_dict)
```

Comment:
BTW should not this be done with condition to run this only when we are running on disconnected cluster?

Reply:
The condition is incorporated into the `update_container_with_mirrored_image()` function: https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/utility/utils.py#L2413

Reply:
After discussion with Daniel I understand that condition is in function laready.

Reply:
Thanks for the review and help :) 
Resolved